src/components/HistoryTable.js

```diff
@@ -196,22 +196,24 @@ class HistoryRow extends PureComponent<HistoryRowProps, HistoryRowState> {
             </a>
           </td>
         </tr>
-        <tr
-          className="history-row-details"
-          style={{ display: busy || open ? "table-row" : "none" }}>
-          <td colSpan="6">
-            {busy ? (
-              <Spinner />
-            ) : previous ? (
-              <Diff source={entry.target} target={previous.target} />
-            ) : error ? (
-              <p className="alert alert-danger">{error.toString()}</p>
-            ) : (
-              <pre>{JSON.stringify(entry.target, null, 2)}</pre>
-            )}
-          </td>
-        </tr>
-      </tbody>
+        {open ? (
```

Comment:
Only add a `<tr>` in the table if it's visible, so we don't mess up the "table stripes" from [bootstrap table stripes](https://getbootstrap.com/docs/4.0/content/tables/#striped-rows)